src/js/program/ad-program-controller.js

```diff
@@ -113,6 +113,7 @@ export default class AdProgramController extends ProgramController {
             event.oldstate = model.get(PLAYER_STATE);
             adMediaModelContext.set('mediaState', event.newstate);
         });
+        this.off();
```

Comment:
`this.off` removes all event listeners, not just 'fullscreenchange' for context `this`.